src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -565,6 +566,12 @@ public static void VerifyWithRevocation()
 
                             Console.WriteLine(
                                 $"{nameof(VerifyWithRevocation)}: online attempt {i} - errors at depth {j}: {allFlags}");
+
+                            if (i > 3)
```

Comment:
Please change this to always sleep if it's going to retry, and not sleep on the last iteration.  If you want to make it check for RevocationStatusUnknown before sleeping, that's fine too.

And please do not increase the retry count.  (If the sleep doesn't help, we can increase the retry count later; but let's not double-slow-things-down).